operations/kyverno/Kyverno.md

```diff
@@ -252,57 +243,55 @@ Also, it generates the report of policy violation in respective workloads. The f
                   privileged: false
     ```
 
-    (`ncn#`) View the policy report status with the following command:
-
-    ```bash
-    kubectl get polr -A
-    ```
-
-    Example output:
+    * (`ncn-mw#`) View the policy report status with the following command:
```

Comment:
Should this be a "-'?

Reply:
I don't see the linter complaining, maybe it doesn't see it or was the linter updated? I'm mobile so it's a little tedious to check.